include/ProgramRepresentation/ProgramPoint.h

```diff
@@ -39,6 +48,11 @@ class ProgramPoint {
     // of these sets
     void addPVAS(PVAliasSet pvas);
 
+    // Returns the return value associated with this program point
+    // Likely will need to be remodeled later so all resources that are owned by
```

Comment:
This kind of comment about how we should change the code in the future should be marked with a "TODO" prefix, so that we can find it later on.